zerver/views/__init__.py

```diff
@@ -572,6 +572,36 @@ def dev_direct_login(request, **kwargs):
     return HttpResponseRedirect("%s%s" % (settings.EXTERNAL_URI_SCHEME,
                                           request.get_host()))
 
+@csrf_exempt
+@require_post
+@has_request_variables
+def api_dev_android_direct_login(request, username=REQ):
```

Comment:
Yep!

-Tim Abbott (mobile)
On Jun 5, 2016 8:24 AM, "Kunal Gupta" <notifications@github.com> wrote:

> In zerver/views/__init__.py
> <https://github.com/zulip/zulip/pull/851#discussion_r65819225>:
>
> > @@ -572,6 +572,36 @@ def dev_direct_login(request, **kwargs):
> >      return HttpResponseRedirect("%s%s" % (settings.EXTERNAL_URI_SCHEME,
> >                                            request.get_host()))
> >
> > +@csrf_exempt
> > +@require_post
> > +@has_request_variables
> > +def api_dev_android_direct_login(request, username=REQ):
>
> It would work If I replace the REQ with REQ() as I see these changes have
> been implemented in this commit 960144a
> <https://github.com/zulip/zulip/commit/960144a49e66bd330daecdf3423e3a53c8353352>
> ,
> So I should just replace those REQ right?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zulip/zulip/pull/851/files/ac7f7be901f77d111751ea635a5f0d2d2dce9b2d#r65819225>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACnm2lHUhVsq0MkQM-_jGCbV4LDojKb1ks5qIuomgaJpZM4IrGW1>
> .
>